db/migrate/20231106131223_change_role_type_in_users.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeRoleTypeInUsers < ActiveRecord::Migration[7.0]
```

Comment:
Role was already integer in db right?

Reply:
yes i thought maybe it isnt, but it was, i will delete the migration